pkgdb2/templates/actions_update.html

```diff
@@ -35,7 +35,12 @@
 <form action="{{ url_for('.admin_action_edit_status', action_id=action_id) }}"
     method="post">
 {% endif %}
-
+{% if is_admin %}
```

Comment:
@pypingou  Do I also have to check if fas user is in admin group? or only this will work?